src/mainwindow.cpp

```diff
@@ -112,83 +113,79 @@ void MainWindow::updateBoms(DbManager *d)
         {
-            count++;
-            if (count == 0)
-                continue;
-
-            if (count == 1)
+            ++count;
+            if (count == 0 || count == 1)
                 continue;
 
-            switch (count % 4)
+            bool ok = false;
+            switch(count % 4)
             {
                 case 2:
-                    firstTeam   =  aString; break;
+                    firstTeam   = list[i]; break;
                 case 3:
-                    firstScore  = atoi(aString.c_str()); break;
+                    firstScore  = list[i].toInt(&ok); break;
                 case 0:
-                    secondScore = atoi(aString.c_str()); break;
+                    secondScore = list[i].toInt(&ok); break;
                 case 1:
-                    secondTeam  =  aString; break;
+                    secondTeam  = list[i]; break;
             }
 
             if ((count % 4) == 1)
             {
-                unsigned int firstTeamInd, secondTeamInd;
-                for (unsigned int i = 0; i < aTeamsId.size(); ++i)
-                {
-                    if (aTeamsId[i]->getSmallName().toStdString() == firstTeam)
-                        firstTeamInd = i+1;
-                    else if (aTeamsId[i]->getSmallName().toStdString() == secondTeam)
-                        secondTeamInd = i+1;
-                }
-
-                std::cout << std::setw(3) << ind1++ << " :"
-                          << std::setw(4) << firstTeam << " (" << std::setw(2) << firstTeamInd;
-                std::cout << ") - ";
-                std::cout << std::setw(4) << secondTeam << " (" << std::setw(2) << secondTeamInd;
-                std::cout << ") / " << firstScore << " - " << secondScore << std::endl; 
+                unsigned int firstTeamInd  = retrieveTeamIndicator(firstTeam);
+                unsigned int secondTeamInd = retrieveTeamIndicator(secondTeam);
+                
+                qDebug() << totalIndicator + 1 << ":"
+                         << firstTeam << " (" << firstTeamInd
+                         << ") - "
+                         << secondTeam << " (" << secondTeamInd
+                         << ") / " << firstScore << " - " << secondScore;
             
                 if (apDbConnection->isOpen())
                 {
-                    if (!apDbConnection->fillMatch(ind2++, firstTeamInd, secondTeamInd, firstScore, secondScore))
-                        std::cout << "Oups..." << std::endl;
+                    if (!apDbConnection->fillMatch(++totalIndicator, firstTeamInd, secondTeamInd, firstScore, secondScore))
+                        stream << "Oups..." << endl;
                 }
             }
         }
     }
-
-    matchFlow.close();
-
+    
+    filePath.close();
     return true;
+}
 
-    /*
-    for (unsigned int i = 1; i < aMatchesId.size(); ++i)
+unsigned int MainWindow::retrieveTeamIndicator(const QString& smallNameToFind)
+{
+    long unsigned int teamSize = aTeamsId.size();
```

Comment:
not long but const ;)

Reply:
After changing all std::vector by QVector, there is no more warning about size() type like "long" unsigned int

Reply:
So modified as asked